torch/_inductor/lowering.py

```diff
@@ -6289,7 +6289,7 @@ def var_mean_sum_(x, axis, correction, keepdim, return_mean):
     return x_var, x_mean
 
 
-def use_two_step_variance(x, axis, keepdim):
+def use_two_step_variance(x, axis, keepdim, threshold=1024):
```

Comment:
how do we get this magic number `1024` here? any benchmarks to demonstrate the the perf benefits and accuracy loss?